cypher/cypher-docs/src/docs/dev/syntax/naming.asciidoc

```diff
@@ -5,22 +5,30 @@ We describe here rules and recommendations for the naming of node labels, relati
 
 == Naming rules
 
-* Must begin with an alphabetic letter.
+* Alphabetic letters:
```

Comment:
Isn't this a tautology? Letters can only be alphabetic. I would rather rephrase it to either:
Alphabetic characters or just Letters.

Reply:
Thank you @renetapopova , you are absolutely right! Addressed in all instances.